stepactions/get-supported-ephemeral-cluster-versions/0.1/README.md

```diff
@@ -0,0 +1,15 @@
+# get-supported-ephemeral-cluster-versions stepaction
+
+This StepAction queries the hub cluster used to provision ephemeral clusters for testing. It returns a list of supported versions stored in a hypershift ConfigMap.
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|kubeconfigSecretLabelSelector|A label selector used to identify the secret containing the EaaS hub cluster kubeconfig|toolchain.dev.openshift.com/spacerequest=eaas|false|
+|insecureSkipTLSVerify|Skip TLS verification when accessing the EaaS hub cluster|false|false|
```

Comment:
I wonder if there are any risks using skipping the verification. if there are, we might want to mention them

Reply:
Added a warning.